proposals/jvm-field-annotation-in-interface-companion.md

```diff
@@ -0,0 +1,42 @@
+# @JvmField for interface companion properties
+
+* **Type**: Design proposal
+* **Author**: Mike Bogdanov
+* **Status**: Implemented in Kotlin 1.3
+* **Discussion**: [KEEP-152](https://github.com/Kotlin/KEEP/issues/152)
+
+## Summary
+
+Allow to use '@JvmField' annotation on interface companion properties similar to class companion ones.
+
+## Description
+
+'@JvmField' annotation on interface companion properties has same effect and similar restrictions as annotation on class companion members:
+* generates static field in interface with initialization in interface <clinit>.
```

Comment:
I think it worth mentioning explicitly that the field will be removed from the companion class, so it's a binary incompatible change for Java clients.